### DISCUSSION TITLE
语音播报

### DISCUSSION THREAD

**Discussion by xfgryujk** [xfgryujk/blivechat#164] [插件]
# 语音播报

用语音读出收到的消息。点击管理打开配置文件，编辑保存后重启插件生效

在Windows上的实现使用了Microsoft Speech API。如果你的默认语音不支持中文，或者想安装新的语音包，可以在`Windows设置 - 时间和语言 - 语音`里安装

## 下载

* [text-to-speech-1.0.1-windows-x64.zip](https://github.com/user-attachments/files/26330240/text-to-speech-1.0.1-windows-x64.zip) SDK v1.0.1
* [text-to-speech-1.0.0-windows-x64.zip](https://github.com/xfgryujk/blivechat/files/14731955/text-to-speech-1.0.0-windows-x64.zip) SDK v1.0.0

### v1.0.1 2026-03-29

* 不播报跨房弹幕
* 升级SDK到v1.0.1

### v1.0.0 2024-03-18

第一个发布版本

**Comment by LeoQuote** [comment]
Windows 10 下面好像不能用, 有没有 debug 的说明?

**Reply by xfgryujk** [reply]
It works on my machine

**Reply by LeoQuote** [reply]
我也不知道怎么解决的, 重新运行了几次又好了

```
[app]
# 语音ID，如果为空则使用默认的。取值看启动时“Available voices:”这条日志里的ID
tts_voice_id = HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Speech\Voices\TokenEnums\NaturalVoiceEnumerator\MicrosoftWindows.Voice.zh-CN.Xiaoxiao.1
# 语速
tts_rate = 200
# 音量
tts_volume = 1.0
```
这是我的配置供有需要的人参考

**Comment by Agnes4m** [comment]
有没有办法选择其他语音

**Comment by Citric-Acid-DC** [comment]
有没有办法播报加入直播间
